### PR TITLE
Creating drop in component for the road label view component.

### PR DESCRIPTION
**Pull request by cafesilencio**
### Description
A component for the drop in UI has been created to represent the road name in the view.

### CODE REVIEW

File: libnavui-dropin/src/main/res/layout/drop_in_state_active_guidance.xml [hunk 1]
```diff
@@ -43,4 +43,13 @@
         app:layout_constraintBottom_toBottomOf="parent"
         />
 
+    <com.mapbox.navigation.ui.maps.roadname.view.MapboxRoadNameView
```
Comment (abhishek1508) [inline libnavui-dropin/src/main/res/layout/drop_in_state_active_guidance.xml:46]: This should be frame layout

File: libnavui-dropin/src/main/res/layout/drop_in_state_free_drive.xml [hunk 2]
```diff
@@ -24,4 +24,13 @@
         app:layout_constraintBottom_toBottomOf="parent"
         android:text="Start Navigation"
         android:background="@color/colorOnPrimary"/>
+
+    <com.mapbox.navigation.ui.maps.roadname.view.MapboxRoadNameView
```
Comment (abhishek1508) [inline libnavui-dropin/src/main/res/layout/drop_in_state_free_drive.xml:28]: This should be frame layout

**Comment by kmadsen** [comment]
@cafesilencio will you add a screen shot?

File: libnavui-dropin/src/main/res/layout/drop_in_state_active_guidance.xml (hunk 1 above)
Comment (cafesilencio) [reply libnavui-dropin/src/main/res/layout/drop_in_state_active_guidance.xml]: changed

File: libnavui-dropin/src/main/res/layout/drop_in_state_free_drive.xml (hunk 2 above)
Comment (cafesilencio) [reply libnavui-dropin/src/main/res/layout/drop_in_state_free_drive.xml]: changed

**Comment by abhishek1508** [comment]
> Screenshot_20220215-105614

@cafesilencio Let us use the exact design as described in [Figma](https://www.figma.com/file/6bXLQGCUaYb7DdJsNL3e0Z/DS-Navigation?node-id=2751%3A28520).

**Comment by cafesilencio** [comment]
> Let us use the exact design as described in [Figma](https://www.figma.com/file/6bXLQGCUaYb7DdJsNL3e0Z/DS-Navigation?node-id=2751%3A28520).   

Firstly I don't have access. Second, I thought it was decided we're using the SDK components and focussing on getting things working rather than making things look pretty.  Our time is better spend on core functionality at this stage rather than cosmetics.

**Comment by abhishek1508** [comment]
>Firstly I don't have access. Second, I thought it was decided we're using the SDK components and focussing on getting things working rather than making things look pretty. Our time is better spend on core functionality at this stage rather than cosmetics.

Anyone at Mapbox has access to this link. Can you please double check?

In case of this PR we are using the component as is. `MapboxRoadNameView` is nothing but a simple text view. All we need to do is create a drawable background in DropInUI and apply it to `MapboxRoadNameView`. Instead of the rectangle in this PR, it just needs to be rounded corners with a border. 

Screen Shot 2022-02-16 at 11 57 27 AM

@cafesilencio

**Review by abhishek1508 — APPROVED** [review]
Thanks for addressing the feedback 🎉 
@cafesilencio

**Pull request merged**